oem/firstvoices/keyboards.csv

```diff
@@ -51,7 +51,7 @@ fv,fv_severn_ojibwa,ᐊᓂᔑᓂᓂᒧᐎᐣ,Eastern Subarctic,fv_severn_ojibwa_
 fv,fv_ojibwa,ᐊᓂᔑᓇᐯᒧᐎᓐ,Eastern Subarctic,fv_ojibwa_kmw-9.0.js
 fv,fv_naskapi,ᓇᔅᑲᐱ,Eastern Subarctic,fv_naskapi_kmw-9.0.js
 sil,sil_euro_latin,English,European,european2-1.6.js
-basic,basic_kbdus,Français,European,canadian_french-1.0.js
+sil,sil_euro_latin,Français,European,canadian_french-1.0.js
```

Comment:
Shouldn't lines 53-54 point to sil_euro_latin-1.8.1.js? (deprecate european2)

Reply:
No, because this is the upgrade pathway for previous users of the app, and the last column indicates the name of the keyboard in the previous release.